src/Command/addBlock.php

```diff
@@ -33,11 +33,15 @@ public function handle()
         $requiresOrdering = array();
         foreach ($installed_modules as $item) {
             if (file_exists($item->path . "/resources/views/" . $this->location . ".twig")) {
+                $order = false;
                 if (isset($this->params['_ORDER_'])) {
                     $order = array_search($item->namespace, $this->params['_ORDER_'], true);
```

Comment:
It is not correct to add module names to base AddBlock. @fatihalp For example, a customer who does not use the customfields module should not see the module name in the codes.

Reply:
Give us a better idea 😉

Reply:
For now, you only use it for filtering.
You can add order parameter to every variable you submit.

maincats : {''data':....,'order': 1}

AdBlock yalnızca kullanıldığı alana etki etmelidir.(ex: ads-list/partials/list-filter only for filter), diğer AdBlock alanları bu alanı başka herhangi bir amaçla kullanmamalıdır.

Reply:
maincats can bu used bu multiple modules, which one of them will be on top then? Not a good idea, come up with a better one.

Reply:
AdBlock must only represent the domain used. Cannot be used in different domains


`{{ addBlock('ads-list/partials/list-filter',{
                            maincats : {''data':....,'order': 1}
                            '...
                        })|raw }}`

That way it only works where you use it.

If this parameter exists, it sorts. If not, it will not find this parameter in addBlock, so the default order will be displayed.

AddBlock only affects where it is used.

If it will make changes to any theme, override already works.